### ISSUE TITLE
Hope to enhance this feature

### ISSUE THREAD

**Issue by zhl111** [zsviczian/obsidian-excalidraw-plugin#298]
Compare diagram and excalidraw these two plug-ins, I think diagram is more convenient and fast, and excalidraw is not very supportive of Chinese typesetting, I do not know if I can refer to diagram? Thank you
![动画](https://user-images.githubusercontent.com/29135984/145682120-2c06a156-2893-4f2d-909d-e5204a2905bb.gif)

**Comment by zsviczian** [comment]
You are absolutely correct. Text wrapping is missing from Excalidraw. I added a few features to help bridge the gap, such as text wrapping for transclusions and markdown embeds... but that does not solve all the issues. This feature needs to be developed in the core Excalidraw product.

That said, you will find some of the scripting functionality available in 1.5.0 helpful. I am just about to release this new update.

**Comment by zhl111** [comment]
Drawing charts is much more efficient, but lacks many of the features of Excalidraw and would love to be able to merge them. Maybe I'm too greedy and really put you to the trouble.
